include/support.h

```diff
@@ -304,6 +304,8 @@ std::vector<std::string> split(const std::string &seq);
 
 bool is_executable_filename(const std::string &filename) noexcept;
 
+bool is_hex_digits(const std::string s) noexcept;
+bool is_digits(const std::string s) noexcept;
```

Comment:
Suggest taking in a `const std::string_view` (or second best, a const references `const std::string &`).

The string_view gives you std::string functionality wrapped around the incoming buffer, without any copying. It handles `const std::string` or `const char *`, and both will be wrapped by `string_view`.

Reply:
Sure, I mostly got into the habit of using references for `std::string` in function args... except when I forget 😂 I'll try `std::string_view`, seems like a good idea and hopefully it will work.

On a second thought, I should move these into `string_utils.h`:

```
commit a8d7a4da63c91d160be4edb6e0d4c84ecf5b3407
Author: Patryk Obara <dreamer.tan@gmail.com>
Date:   Sat Nov 14 05:11:54 2020 +0100

    Move starts_with function to a new header

    New header is introduced to gradually move our new string-related helper
    functions in there (out of support header).
```